rest-api-spec/test/cluster.put_settings/10_basic.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 setup:
   - skip:
-      version: 0 - 999
+      version:  " - "
```

Comment:
I don't think `" - "` is an improvement over `"0 - 999"`. I'd rather see some meaningful value, like `all`?